### PR TITLE
Area Annotation Toolbox GUI

### PR DESCRIPTION
**Pull request by yinghj**
Deprecate the onGui UI drawing methods for area annotation toolbox, replacing it with UI elements on Canvas in scene.

### CODE REVIEW

File: Project/Assets/Prairie/Framework/Script/Annotation/AreaAnnotationGui.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	public FirstPersonInteractor FPI;
+
+	// Name of the areaAnnotationGUI object in game scene.
+	private const string AREAANNOTATION = "AnnotationPanel";
+
+	void Start() {
+		gameObject.transform.Find(AREAANNOTATION).gameObject.SetActive (active);
+	}
+
+	public void ActivateGui() {
```
Comment (Charlieranderson) [inline Project/Assets/Prairie/Framework/Script/Annotation/AreaAnnotationGui.cs]: Could you maybe add some comments on these small methods just detailing where and when they might be called? I can't tell just from looking at this code.

**Review by Charlieranderson — COMMENTED** [review]
Looks good except for the one change I asked for.

**Comment by yinghj** [comment]
Also added comments that addressed Charlie's request.

**Review by Charlieranderson — APPROVED** [review]

**Comment by yinghj** [comment]
Do NOT merge. Discovered new bug when moving from one set of overlapping area annotation colliders to another.

**Comment by Charlieranderson** [comment]
Is the bug fixed now?

**Comment by yinghj** [comment]
Yes. This is ready for review and mergeable now.

**Review by adanter — APPROVED** [review]
Looks good to me

**Pull request merged**